ibis/backends/dask/execution/util.py

```diff
@@ -218,29 +223,65 @@ def compute_sort_key(
     name = ibis.util.guid()
     if key.name in data:
         return name, data[key.name]
+    if isinstance(key, str):
+        return key, None
+    elif key.name in data:
+        return key.name, None
     else:
         if scope is None:
             scope = Scope()
         scope = scope.merge_scopes(
             Scope({t: data}, timecontext) for t in an.find_immediate_parent_tables(key)
         )
         new_column = execute(key, scope=scope, **kwargs)
+        name = ibis.util.guid()
```

Comment:
`name`. should already be a guid from line 223? https://github.com/ibis-project/ibis/pull/5747/files#diff-7560a298b0242d66bc062059c8a1189235eea9088c8174ce22b7154caf1d629fR223

Reply:
Yup, you're right here. It's duplicated and not needed